{'activation': 'linear', 'dropout': 0.3231506876, 'hist_agg': 'none', 'layers': 1, 'learning_method': 'rmsprop', 'lr': 0.0090546172, 'n_cases': 16651, 'n_features': 311, 'n_games': 5, 'steps': 300, 'units': 48}
('Failed to impute column p.none.5.loss:1. Only nans in the column and no imputation data provided', 'p.none.5.loss:1', ValueError('shape mismatch: value array of shape (0,215758) could not be broadcast to indexing result of shape (1,215758)',))
['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 21, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 791, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 770, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


{'activation': 'tanh', 'dropout': 0.3297911725, 'hist_agg': 'none', 'layers': 2, 'learning_method': 'rmsprop', 'lr': 0.0099025046, 'n_cases': 61676, 'n_features': 249, 'n_games': 4, 'steps': 100, 'units': 84}
('Failed to impute column p.none.4.loss:1. Only nans in the column and no imputation data provided', 'p.none.4.loss:1', ValueError('shape mismatch: value array of shape (0,217086) could not be broadcast to indexing result of shape (1,217086)',))
['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 21, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 791, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 770, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


{'activation': 'relu', 'dropout': 0.3185924675, 'hist_agg': 'none', 'layers': 2, 'learning_method': 'rmsprop', 'lr': 0.0054918231, 'n_cases': 67706, 'n_features': 125, 'n_games': 2, 'steps': 600, 'units': 55}
('Failed to impute column p.none.2.loss:1. Only nans in the column and no imputation data provided', 'p.none.2.loss:1', ValueError('shape mismatch: value array of shape (0,219828) could not be broadcast to indexing result of shape (1,219828)',))
['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 21, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 791, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 770, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


{'activation': 'sigmoid', 'dropout': 0.5178580289, 'hist_agg': 'none', 'layers': 2, 'learning_method': 'rmsprop', 'lr': 0.0062466347, 'n_cases': 55700, 'n_features': 249, 'n_games': 4, 'steps': 700, 'units': 31}

['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 21, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 791, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 770, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


